src/components/Navbar.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import {Link,NavLink} from "react-router-dom"
+
+class Navbar extends React.Component{
+    constructor (props){
+        super(props)
+        this.state ={}
+    }
+
+    render(){
+        return(
+    <nav className='navbar'>
+        <div>
+        <Link to="/">TodoApps </Link>
+        </div>
+      <ul>
+        <li> <NavLink to ="/"> Home</NavLink></li>
```

Comment:
This Navlink is rather called **todos** suggestive of the functionality of the components